google-compute-lib.js

```diff
@@ -0,0 +1,219 @@
+/* eslint no-use-before-define: 0 */
+
+const gcpCompute = require("@google-cloud/compute");
+const _ = require("lodash");
+require("@google-cloud/compute");
+const { ProjectsClient } = require("@google-cloud/resource-manager");
+const GCCompute = require("@google-cloud/compute");
+const parsers = require("./parsers");
+const {
+  getParseFromParam,
+  mapAutoParams,
+  getCredentials,
+  getProject,
+  getRegion,
+  getZone,
+} = require("./helpers");
```

Comment:
Importing `Helpers` object instead of destructuring it to get functions, would differentiate these imported functions' calls from the rest of the functions declared in this file.

Reply:
done